File: src/com/lannstark/lec01/Person.java
package com.lannstark.lec01;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class Person {
    String name;

    Person(String name) {
        this.name = name;
    }

        @NotNull
    //@Nullable
    public String getName() {
        return name;
    }
}

File: src/com/lannstark/lec01/Person.kt
package com.lannstark.lec01

class Person(name: String) {
   private var name = name;

    // # 생성자 사용
/*  constructor(name: String)
    {
        this.name = name;
    }*/

    fun  GetName() : String
    {
        return this.name;
    }
}